zproject/urls.py

```diff
@@ -188,7 +188,12 @@
 
     # Returns a 204, used by desktop app to verify connectivity status
     url(r'generate_204$', 'generate_204'),
-
+) + patterns('zerver.views.realm_emoji',
+    url(r'^realm/emoji$', 'rest_dispatch',
+        {'GET': 'list_emoji',
+         'PUT': 'upload_emoji'}),
+    url(r'^realm/emoji/(?P<emoji_name>[a-zA-Z0-9.-]+)$', 'rest_dispatch',
```

Comment:
So this forbids `_` while the `zerver/models.py` implementation allows it -- is that a bug here?